backend-api/infra/monitoring/alarms.yaml

```
@@ -2196,3 +2196,32 @@ Resources:
       Threshold: 25
       ComparisonOperator: GreaterThanOrEqualToThreshold
       TreatMissingData: notBreaching
+
+  AsyncIssueBiometricCredentialZeroVcsIssuedAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: DeployAlarms
+    DependsOn:
+      - "AsyncIssueBiometricCredentialMetricFilter"
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:platform-alarms-sns-warning"
+      OKActions:
+        - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:platform-alarms-sns-warning"
+      InsufficientDataActions: []
+      AlarmDescription: !Sub
+        - "Alarms when there are 0 log events with the messageCode MOBILE_ASYNC_ISSUE_BIOMETRIC_CREDENTIAL_VC_ISSUED within a set timeframe. See runbook: ${RunbookUrl}"
```

"There have been 0 VCs issued in the past 20 minutes. See runbook: ${RunbookUrl}"
```suggestion
        - "Alarms when there are 0 log events with the messageCode MOBILE_ASYNC_ISSUE_BIOMETRIC_CREDENTIAL_VC_ISSUED within a set timeframe. See runbook: ${RunbookUrl}"
```